### ISSUE TITLE
How to persist data in Docker image

### ISSUE THREAD

**Issue by nicholasamorim** [OpenIdentityPlatform/OpenAM#213]
I've been using the docker image but I do not understand how to persist data. There's nothing in `/home/openam/` or `/root` to persist.

What folder should I persist in order to have the configuration survive across reboots?

**Comment by maximthomas** [comment]
OpenAM configuration directory by default is in `/usr/openam/config` folder, you can mount it as a volume for persistence. You can also change the directory via `$OPENAM_DATA_DIR` env variable.
I'll update quick start manual to make it clear

**Comment by nicholasamorim** [comment]
Simply spinning a container with `/usr/openam/config` gives this error when setting up initial configuration:


`/usr/openam/config/opends/opendj.zip (No such file or directory), refer to install.log under /usr/openam/config for more information.`

Seems to expect something to exist inside the folder. And if we're mounting it as a volume it will be initially empty.

**Comment by maximthomas** [comment]
Ok, I'll try to figure out the cause of the problem as soon as I can.

**Comment by maximthomas** [comment]
@nicholasamorim 
Hi, I can't reproduce the issue, created simple `docker-compose.yaml` file started with `docker-compose up` command, configured OpenAM, restarted container, and everything still works fine
```yaml
version: '3'
services:
  openam:
    image: openidentityplatform/openam
    ports:
      - "8080:8080"
    hostname: openam
    volumes:
      - ./openam-data:/usr/openam/config
```

**Comment by mtruneck** [comment]
I just ran into the same problem and want to share what was the cause on my side.

It worked for me on Mac because docker volumes have their own permissions inside of the containers. On Linux, the permissions stay the same as the source directory on host, so I had to manually change owner uid to 1001:1001 (which is the uid of openam inside the contaier).

So `chown 1001:1001 -R openam-data` solved the problem.
